frontend/src/pages/CreateNewInquiry.js

```diff
@@ -1,7 +1,9 @@
 import FormReactHookCreateNewInquiry from "../components/FormReactHookCreateNewInquiry";
 import {getInquiries, postInquiry} from "../services/inquiryService";
-import {useEffect, useState} from "react";
+import React, {useEffect, useState} from "react";
```

Comment:
this looks incomplete - did you want to include `React` within the brackets {}?

Reply:
Good Point !